apps/dataflow/src/scraper/fms/driver.py

```
@@ -0,0 +1,226 @@
+    wait: WebDriverWait,
+    driver: webdriver.Chrome,
+    building_label: str,
+) -> None:
+    """Click a building and all its floors."""
+    global building_count, floor_count  # noqa: PLW0603
+
+    floor_links = wait.until(
+        ec.presence_of_all_elements_located((
+            By.XPATH,
+            # Find floors for this building in the nav tree
+            f"//a[contains(normalize-space(.), '{building_label}')]"  # noqa: E501
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (unused: `E501`)

Remove unused `noqa` directive

(RUF100)